src/css/index.css

```diff
@@ -11,7 +11,7 @@ body {
   .container__dark-theme-calculator {
     width: 375px;
     height: 812px;
```

Comment:
Open your project on an external monitor and on the laptop's screen and try to see a difference.
After that, you need to read about min-height, max-height, min-width, and max-width in CSS. Also, you need to read 1 or 2 articles about responsive markup. Just one or two. After that try to make your app responsive. Firstly you need to make it responsive only for laptops and big screens. Not for mobiles. The mobile responsive design we will do in the end.